qiskit_ibm/api/rest/runtime.py

```diff
@@ -95,6 +95,7 @@ def create_program(
         """
         url = self.get_url('programs')
         data = {'name': name,
+                'data': program_data,
```

Comment:
The new `data` field needs to be a string, base64 encoded. See https://runtime-us-east-dev.quantum-computing.ibm.com/openapi/#/Programs/create_program